docs/src/components/ComponentComparison/styles.module.css

```
@@ -0,0 +1,217 @@
+  background: var(--ifm-background-surface-color);
+  padding-top: 0.75rem;
+  border-bottom: 1px solid var(--ifm-color-emphasis-300);
+}
+
+.dataRow:hover th,
+.dataRow:hover td {
+  background: var(--ifm-color-emphasis-100);
+}
+
+.cell {
+}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)